modules/file-system/netapp-volume/outputs.tf

```diff
@@ -0,0 +1,66 @@
+# Copyright 2025 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+output "network_storage" {
+  description = "Describes a NetApp Volumes volume."
+  value = {
+    server_ip             = local.server_ip
+    remote_mount          = local.remote_mount
+    local_mount           = var.local_mount
+    fs_type               = local.fs_type
+    mount_options         = local.mount_options
+    client_install_runner = local.install_nfs_client_runner
+    mount_runner          = local.mount_runner
+  }
+}
+
+output "install_nfs_client" {
+  description = "Script for installing NFS client"
+  value       = file("${path.module}/scripts/install-nfs-client.sh")
+}
+
+output "install_nfs_client_runner" {
+  description = "Runner to install NFS client using the startup-script module"
+  value       = local.install_nfs_client_runner
+}
+
+output "mount_runner" {
+  description = <<-EOT
+  Runner to mount the file-system using an ansible playbook. The startup-script
+  module will automatically handle installation of ansible.
+  - id: example-startup-script
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - $(your-fs-id.mount_runner)
+  ...
+  EOT
+  value       = local.mount_runner
```

Comment:
Is this mount option supposed to be used in vm startup script ? I didn't see this in the example.

Reply:
I am adding the volume to SLURM using the USE directive. This works fine. A few weeks ago, I was able to use a tweaked mount.sh to pass multiple IPs and mount round robin. But that stopped working. I investigated and it turned out that mount.sh was not being used at all anymore. I plan to investigate how the mounts are happening now and how i can insert doing round-robin mount of multiple NFS server IPs, but for now my goal is to get this work here finished and then built an addition on top of it.
Thank you for the information you provided!